pkg/activator/net/revision_backends_test.go

```diff
@@ -250,17 +250,17 @@ func TestRevisionWatcher(t *testing.T) {
 		ticks:     []time.Time{time.Now(), time.Now()},
 		updateCnt: 3,
 	}} {
-
 		t.Run(tc.name, func(t *testing.T) {
+			defer ClearAll()
 			fakeRt := activatortest.FakeRoundTripper{
 				ExpectHost:         "test-revision",
 				ProbeHostResponses: tc.probeHostResponses,
 				ProbeResponses:     tc.probeResponses,
 			}
 			rt := network.RoundTripperFunc(fakeRt.RT)
 
-			updateCh := make(chan *RevisionDestsUpdate, 100)
-			tickerCh := make(chan time.Time, 1)
+			updateCh := make(chan *RevisionDestsUpdate, len(tc.ticks))
```

Comment:
I think this should be tc.ticks + 1 as the dests write will also cause an update.

Reply:
Done.